packages/mobile/src/app/saga.ts

```diff
@@ -130,6 +152,9 @@ export function* handleDeepLink(action: OpenDeepLink) {
       navigate(Screens.FiatExchangeOptions, { isAddFunds: true })
     } else if (rawParams.pathname === '/bidali') {
       navigate(Screens.BidaliScreen, { currency: CURRENCY_ENUM.DOLLAR })
+    } else if (isSecureOrigin && rawParams.path.startsWith('/openScreen') && rawParams.query) {
+      const params = parseQuery(rawParams.query)
+      navigate(params['screen'] as keyof StackParamList, params)
```

Comment:
Can you add a comment that this is limited to our internal usage? and should NOT be exposed publicly for security reasons.